project_national/sdr_upgrades_tmy3.yml

```diff
@@ -399,28 +499,159 @@ references:
+    - or:
+        - not:
+          - or:
+            - HVAC Shared Efficiencies|Boiler Baseboards Heating Only, Electricity
+            - HVAC Shared Efficiencies|Boiler Baseboards Heating Only, Fuel
+            - HVAC Shared Efficiencies|Fan Coil Heating and Cooling, Electricity
+            - HVAC Shared Efficiencies|Fan Coil Heating and Cooling, Fuel
+            - HVAC Shared Efficiencies|Fan Coil Cooling Only
+        # - or:
+        - and:
+          - HVAC Cooling Type|Ducted Heat Pump
+          - or:
+            - Geometry Building Type ACS|10 to 19 Unit
+            - Geometry Building Type ACS|2 Unit
+            - Geometry Building Type ACS|20 to 49 Unit
+            - Geometry Building Type ACS|3 or 4 Unit
+            - Geometry Building Type ACS|5 to 9 Unit
+            - Geometry Building Type ACS|50 or more Unit
+        - and:
+          - HVAC Cooling Type|Non-Ducted Heat Pump
+          - or:
+            - Geometry Building Type ACS|10 to 19 Unit
+            - Geometry Building Type ACS|2 Unit
+            - Geometry Building Type ACS|20 to 49 Unit
+            - Geometry Building Type ACS|3 or 4 Unit
+            - Geometry Building Type ACS|5 to 9 Unit
+            - Geometry Building Type ACS|50 or more Unit
+        - and:
+          - HVAC Cooling Type|Central AC
+          - not: HVAC Shared Efficiencies|Fan Coil Cooling Only
+          - or:
+            - Geometry Building Type ACS|10 to 19 Unit
+            - Geometry Building Type ACS|2 Unit
+            - Geometry Building Type ACS|20 to 49 Unit
+            - Geometry Building Type ACS|3 or 4 Unit
+            - Geometry Building Type ACS|5 to 9 Unit
+            - Geometry Building Type ACS|50 or more Unit
+
+  dual_fuel_cold_apply_logic: &dual_fuel_system_cold_climate
+    - HVAC Has Ducts|Yes # apply to homes with ducts who have non-shared HVAC or shared HVAC with room for a condenser
+    - not:
+      - or:
+        - HVAC Shared Efficiencies|Boiler Baseboards Heating Only, Electricity
+        - HVAC Shared Efficiencies|Boiler Baseboards Heating Only, Fuel
+        - HVAC Shared Efficiencies|Fan Coil Heating and Cooling, Electricity
+        - HVAC Shared Efficiencies|Fan Coil Heating and Cooling, Fuel
+        - HVAC Shared Efficiencies|Fan Coil Cooling Only
+    - or:
+      - ASHRAE IECC Climate Zone 2004|5A
+      - ASHRAE IECC Climate Zone 2004|5B
+      - ASHRAE IECC Climate Zone 2004|6A
+      - ASHRAE IECC Climate Zone 2004|6B
+      - ASHRAE IECC Climate Zone 2004|7A
+      - ASHRAE IECC Climate Zone 2004|7B
+      - ASHRAE IECC Climate Zone 2004|7AK
+      - ASHRAE IECC Climate Zone 2004|8AK
+    - or:
+      - Heating Fuel|Natural Gas
+      - Water Heater Fuel|Natural Gas
+      - Clothes Dryer|Gas, Premium
+      - Clothes Dryer|Gas
+      - Cooking Range|Gas
+      - HVAC Secondary Heating Fuel|Natural Gas
+      - Misc Pool Heater|Natural Gas
+      - Misc Hot Tub Spa|Natural Gas
+      - not: Misc Gas Lighting|None
```

Comment:
@whiphi92  - Misc Gas <item> should be removed. They are randomly assigned in ResStock and could overestimate the # of homes with gas access.